github-actions-runner/Dockerfile

```diff
@@ -1,9 +1,16 @@
-FROM quay.io/evryfs/github-actions-runner:ubuntu20-20201210.0-2.277.1
+FROM quay.io/evryfs/github-actions-runner:ubuntu20-20201210.0-2.275.1
 
 USER root
-RUN apt-get update && \
+RUN sudo apt-key adv --refresh-keys --keyserver keyserver.ubuntu.com && \
+    apt-get update && \
     apt-get -y install gcc libpython3-dev && \
     apt-get -y clean && \
     rm -rf /var/cache/apt /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Update latest runner
+ARG RUNNER_VERSION=2.277.1
```

Comment:
Looks good, what I'm wondering is how do we intend to keep this updating automatic?

Reply:
Working on it :-)

Reply:
So I've done it like this: https://github.com/ebi-gene-expression-group/atlas-containers/commit/32687f5b58d9b7a9b6259840fdc04860d48d3f3b

Basically, extract the upstream tag minus the version, then append the explicit RUNNER_VERSION.